lib/Bagger/Agent/Storage/Mapper.pm

```diff
@@ -78,7 +78,7 @@ my %keygen = (
 
 my %classmap = (
     'Bagger::Storage::Config'       => 'config',
-    'Bagger::Storage::Instance'     => 'postgres_instances',
+    'Bagger::Storage::Instance'     => 'postgres_instance',
```

Comment:
Inconsistency in singular vs plural names is a big reason to push for the standardization on singular.